<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgCursos</class>
 <widget class="QDialog" name="dlgCursos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>295</width>
    <height>412</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>295</width>
    <height>412</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>295</width>
    <height>412</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Configuración de Cursos</string>
  </property>
  <widget class="QGroupBox" name="gbTManiana">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>231</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Turno Mañana</string>
   </property>
   <widget class="QLabel" name="lblNivel1M">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nivel 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblNivel2M">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nivel 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblNivel3M">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nivel 3</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbM1A">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbM2A">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbM3A">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbM1B">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbM3B">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbM2B">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbM1C">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbM3C">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbM2C">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pbGuardarCursos">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>360</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Guardar Configuración del año</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gbTTarde">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>231</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Turno Tarde</string>
   </property>
   <widget class="QLabel" name="lblNivel1T">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nivel 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblNivel2T">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nivel 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblNivel3T">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nivel 3</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbT1A">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbT2A">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbT3A">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbT1B">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbT3B">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbT2B">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbT1C">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbT3C">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbT2C">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="cbboxAnio">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblAnio">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Año</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
